scripts/Synapse/analyzeLinkageAlgorithms.ipynb

```diff
@@ -241,7 +349,8 @@
         "extracted_patient_data = extracted_patient_data.join(full_name_table, \"patient_id\", \"left\")\n",
         "extracted_patient_data.cache()\n",
         "\n",
-        "# Identifier table needs compiled and can left join back to patients\n",
+        "# Identifier table needs compiled (patients can have multiple IDs, such as MRN and\n",
+        "# SS) and can left join back to patients\n",
         "extracted_identifier_data = spark.sql(f\"SELECT * from {DB_TABLE_IDENTIFIER}_view\")\n",
```

Comment:
I had thought of that, but I wanted to leave the option in in case LAC wanted to work with SSN on their own side. As this is written, we just extract any and all identifiers that are present, so those could be local codes or something related to providers/hospitals as well. I think it's unlikely to come up, but at least this way they theoretically can work with any provided identifier instead of just MRN.